extensions/identity-hub/src/test/java/org/eclipse/dataspaceconnector/identityhub/api/IdentityHubControllerTest.java

```diff
@@ -45,7 +45,7 @@
 @ExtendWith(EdcExtension.class)
 public class IdentityHubControllerTest {
 
-    private static final int PORT = getFreePort();
+    private static final int PORT = TestUtils.getFreePort();
```

Comment:
very nit: this change needed ? can work as static import as before ? Just pointing it out as it might be side-effect of other changes.

Reply:
It's indeed a side effect. I bring it back how it was.